### ISSUE TITLE
Runtime support for Java 11

### ISSUE THREAD

**Issue by NicolasRouquette** [scalacenter/scalafix#985]
As mentioned in the JDK compatiblity notes (https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html), it is important to support JDK 11 since it is an LTS release.

As far as scalafix is concerned, there is at lease an issue in `ClasspathOps` due to casts to `URLClassLoader`, e.g.:

```scala
  def thisClassLoader: URLClassLoader =
    this.getClass.getClassLoader.asInstanceOf[URLClassLoader]
```

This is one of the known migration problems described here:
https://blog.codefx.org/java/java-11-migration-guide/#Casting-To-URL-Class-Loader

There are some possible workarounds:
http://java9.wtf/class-loading/
https://stackoverflow.com/questions/46694600/java-9-compatability-issue-with-classloader-getsystemclassloader

I wonder if it is necessary to have JDK-specific `ClasspathOps` code to handle the differences in JDK8 vs. JDK9+.

**Comment by RicoGit** [comment]
Any progress about this? Docs says that java11 is supported.
https://scalacenter.github.io/scalafix/docs/users/installation.html

**Comment by olafurpg** [comment]
The JDK 11 issues should be fixed in the latest release v0.9.13. Our CI runs tests on JDK 11. Please open a new ticket if the problem persists
